graalpython/lib-graalpython/modules/ginstall.py

```diff
@@ -555,7 +555,13 @@ def main(argv):
                 info(p[len(user_site) + 1:])
     elif args.command == "uninstall":
         warn("WARNING: I will only delete the package folder, proper uninstallation is not supported at this time.")
-        user_site = site.getusersitepackages()
+        if site.ENABLE_USER_SITE:
+            user_site = site.getusersitepackages()
+        else:
+            for s in site.getsitepackages():
+                if s.endswith("site-packages"):
+                    user_site = s
+                    break
```

Comment:
Could you please factor out the common part that you copied from "list" to a separate function?

Reply:
@msimacek I'm done!